Comparing source compatibility of /Users/bluefire/git/uima-uimaj/target/checkout/uimaj-core/target/uima-core.jar against /Users/bluefire/.m2/repository/org/apache/uima/uimaj-core/3.2.0/uimaj-core-3.2.0.jar
WARNING: You have ignored certain classes, i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.analysis_component.CasProcessor  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.analysis_component.CasProcessor andThen(org.apache.uima.analysis_component.CasProcessor)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void process(org.apache.uima.cas.CAS)
		+++  NEW EXCEPTION: java.lang.Throwable
	+++  NEW ANNOTATION: java.lang.FunctionalInterface
+++* NEW CLASS: PUBLIC(+) org.apache.uima.analysis_component.CasProcessorAnnotator  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW INTERFACE: org.apache.uima.analysis_engine.TextAnalysisEngine
	+++  NEW INTERFACE: org.apache.uima.collection.CasConsumer
	+++  NEW INTERFACE: org.apache.uima.collection.base_cpm.CasObjectProcessor
	+++  NEW INTERFACE: org.apache.uima.analysis_engine.AnalysisEngine
	+++  NEW INTERFACE: org.apache.uima.resource.Resource
	+++  NEW INTERFACE: org.apache.uima.collection.base_cpm.CasProcessor
	+++  NEW INTERFACE: org.apache.uima.resource.ConfigurableResource
	+++  NEW SUPERCLASS: org.apache.uima.analysis_engine.impl.AnalysisEngineProcessorAdapter
	+++  NEW CONSTRUCTOR: PUBLIC(+) CasProcessorAnnotator(org.apache.uima.analysis_component.CasProcessor)
	+++  NEW METHOD: PUBLIC(+) boolean initialize(org.apache.uima.resource.ResourceSpecifier, java.util.Map)
		+++  NEW EXCEPTION: org.apache.uima.resource.ResourceInitializationException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.analysis_component.CasProcessorAnnotator of(org.apache.uima.analysis_component.CasProcessor)
		+++  NEW EXCEPTION: org.apache.uima.resource.ResourceInitializationException
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.uima.analysis_component.JCasProcessor  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) org.apache.uima.analysis_component.JCasProcessor andThen(org.apache.uima.analysis_component.JCasProcessor)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void process(org.apache.uima.jcas.JCas)
		+++  NEW EXCEPTION: java.lang.Throwable
	+++  NEW ANNOTATION: java.lang.FunctionalInterface
+++* NEW CLASS: PUBLIC(+) org.apache.uima.analysis_component.JCasProcessorAnnotator  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW INTERFACE: org.apache.uima.analysis_engine.TextAnalysisEngine
	+++  NEW INTERFACE: org.apache.uima.collection.CasConsumer
	+++  NEW INTERFACE: org.apache.uima.collection.base_cpm.CasObjectProcessor
	+++  NEW INTERFACE: org.apache.uima.analysis_engine.AnalysisEngine
	+++  NEW INTERFACE: org.apache.uima.resource.Resource
	+++  NEW INTERFACE: org.apache.uima.collection.base_cpm.CasProcessor
	+++  NEW INTERFACE: org.apache.uima.resource.ConfigurableResource
	+++  NEW SUPERCLASS: org.apache.uima.analysis_engine.impl.AnalysisEngineProcessorAdapter
	+++  NEW CONSTRUCTOR: PUBLIC(+) JCasProcessorAnnotator(org.apache.uima.analysis_component.JCasProcessor)
	+++  NEW METHOD: PUBLIC(+) boolean initialize(org.apache.uima.resource.ResourceSpecifier, java.util.Map)
		+++  NEW EXCEPTION: org.apache.uima.resource.ResourceInitializationException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.uima.analysis_component.JCasProcessorAnnotator of(org.apache.uima.analysis_component.JCasProcessor)
		+++  NEW EXCEPTION: org.apache.uima.resource.ResourceInitializationException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.analysis_engine.AnalysisEngine  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PARAM_CONFIG_PARAM_SETTINGS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PARAM_RESOURCE_MANAGER
		+++  NEW ANNOTATION: java.lang.Deprecated
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.analysis_engine.AnalysisEngineServiceStub  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===! UNCHANGED INTERFACE: org.apache.uima.resource.ResourceServiceStub
	***! MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void callBatchProcessComplete()
		===  UNCHANGED EXCEPTION: org.apache.uima.resource.ResourceServiceException
	***! MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void callCollectionProcessComplete()
		===  UNCHANGED EXCEPTION: org.apache.uima.resource.ResourceServiceException
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.apache.uima.Constants  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROTOCOL_SOAP_WITH_ATTACHMENTS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROTOCOL_SOAP
		+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.metadata.ConfigurationParameter  (default serialVersionUID changed)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String TYPE_DOUBLE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String TYPE_LONG
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.metadata.FsIndexCollection  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void resolveImports(java.util.Collection, org.apache.uima.resource.ResourceManager)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: org.apache.uima.util.InvalidXMLException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.metadata.TypePriorities  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void resolveImports(java.util.Collection, org.apache.uima.resource.ResourceManager)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: org.apache.uima.util.InvalidXMLException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.metadata.TypeSystemDescription  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void resolveImports(java.util.Collection, org.apache.uima.resource.ResourceManager)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: org.apache.uima.util.InvalidXMLException
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.RelativePathResolver  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getDataPath()
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.List getDataPathElements()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setDataPath(java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setDataPathElements(java.lang.String[])
		+++  NEW EXCEPTION: java.net.MalformedURLException
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setDataPathElements(java.io.File[])
		+++  NEW EXCEPTION: java.net.MalformedURLException
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.ResourceManager  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getDataPath()
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.List getDataPathElements()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setDataPath(java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.net.MalformedURLException
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setDataPathElements(java.lang.String[])
		+++  NEW EXCEPTION: java.net.MalformedURLException
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setDataPathElements(java.io.File[])
		+++  NEW EXCEPTION: java.net.MalformedURLException
===* UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.ResourceManagerPearWrapper  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===* UNCHANGED INTERFACE: org.apache.uima.resource.ResourceManager
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.ResourceServiceStub  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	***! MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) void destroy()
***! MODIFIED CLASS: PUBLIC org.apache.uima.UimaContextHolder  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	***! MODIFIED CONSTRUCTOR: PRIVATE (<- PUBLIC) UimaContextHolder()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.util.Logger  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setResourceManager(org.apache.uima.resource.ResourceManager)
		+++  NEW ANNOTATION: java.lang.Deprecated
